Data-Structures/src/main/java/Graph/Graph.java

```diff
@@ -1,4 +1,191 @@
 package Graph;
 
+import java.util.*;
+
 public class Graph {
+
+    public List<Vertex> Vertices;
```

Comment:
Variable name should start with a lowercase letter -> `vertices`

Reply:
yeah forgot to change that...